package.json

```diff
@@ -54,6 +54,7 @@
     "eslint-plugin-react": "^7.0.1",
     "express": "^4.16.2",
     "faye-websocket": "^0.11.3",
+    "got-scraping": "^3.2.4-beta.0",
```

Comment:
Btw, why got-scraping? I can't find it anywhere else in the changes.

Reply:
test/server.js lines 487-551

> why got-scraping?

`request` proxy implementation fails to send proper IPv6 target sends `...` instead of `[...]`

Reply:
Ah, thanks. The diff was collapsed so cmd+F failed me :)

Reply:
Hint: press <kbd>.</kbd> :D

Reply:
Although the search (sidebar) feature is quite broken, instead of searching all files it just searches through the current file :( So need to browser file by file.